src/components/Editor/Editor.css

```diff
@@ -86,6 +86,11 @@ html[dir="rtl"] .editor-mount {
   right: -4px;
 }
 
+.editor.new-breakpoint {
+  width: 29px;
+  position: absolute;
+}
```

Comment:
We should figure out why the gutter is set to width `1px` https://github.com/codemirror/CodeMirror/blob/master/src/display/update_line.js#L112-L121

My guess is that we're updating the gutter while the element is detatched or the document is not visible.